src/Module.ts

```diff
@@ -40,6 +39,7 @@ import relativeId from './utils/relativeId';
 import { RenderOptions } from './utils/renderHelpers';
 import { SOURCEMAPPING_URL_RE } from './utils/sourceMappingURL';
 import { timeEnd, timeStart } from './utils/timers';
+import { handleMissingExport } from './utils/defaults';
```

Comment:
Is it possible that you are not running the commit hooks when committing? Because otherwise, this import should have been placed at the exact same position it was before your modifications. I also get a TSLint error for this line.

Reply:
It could also just be the case that you need to do an `npm install`.